content/docs/dvclive/api-reference/live/log_params.md

```diff
@@ -0,0 +1,54 @@
+# Live.log_params()
+
+Logs the given group of parameters `params`, saving them as yaml.
+
+```py
+ def log_params(params: Dict[ParamLike]):
+```
+
+Also see `Live.log_param()`.
+
+## Usage
+
+```py
+from dvclive import Live
+
+live = Live()
+
+params = {
+    "lr": 1e-3,
+    "n_iter": 50
+}
+live.log_params(params)
+```
+
+## Description
+
+On each `live.log_params(params)` call, DVCLive will write keys/values pairs in
+the `params` dict to `{Live.params_file}`:
+
+```dvc
+$ cat dvclive/params.yaml
+lr: 1e-3
+n_iter: 50
+```
+
+<admon type="tip">
+
+The parameters file `{Live.params_path}` is usable by `dvc params`,
+`dvc metrics` and `dvc exp show`/`dvc exp diff`.
+
+</admon>
+
+## Parameters
+
+- `params` - Dictionary with name/value pairs of parameters to be logged.
+
+## Exceptions
+
+- `dvclive.error.InvalidParameterType` - thrown if any of the parameter values
+  in the `params` dict is not among supported types. Supported types include:
+  `{ParamLike=}`.
```

Comment:
Seems like the ParamLike definition is missing here?

Reply:
Added the definition, although I'm not sure it's worth it to have the exact definition in the docs. Maybe it will be enough to have something like:

> Supported types include basic python data types such as int, float, bool as well as maps and lists.